spec/actors/hyrax/actors/generic_work_actor_spec.rb

```diff
@@ -242,6 +236,7 @@
         expect(old_parent.reload.members).to eq []
       end
     end
+
```

Comment:
I think the breaks between context blocks leads to readability.  Do you feel strongly that  we need to compress everything?